### ISSUE TITLE
Mod loads on Vanilla servers.

### ISSUE THREAD

**Issue by AlvarOsO** [cyanobacterium/Minecraft_Mineralogy#6]
In 2.6.0 on 1.8.9-11.15.1.1722 Forge, if you enter a Vanilla or Forge server without the mod, the mod loads anyway.
It is obvious because it uses the mod Ore Renderer:
Vanilla server with the mod in the client:
![Vanilla server with the mod](https://cloud.githubusercontent.com/assets/559819/13204249/64035fa8-d8cb-11e5-8e30-fecf0aea0349.png)
Vanilla server on client 1.8.9:
![Vanilla server on client 1.8.9](https://cloud.githubusercontent.com/assets/559819/13204252/790c76d2-d8cb-11e5-85e6-909d6791c90e.png)

**Comment by cyanobacterium** [comment]
Mods load before the game starts and cannot unload at runtime. Minecraft Mineralogy therefore cannot know while it is loading whether you intend to join a server that does not have Minecraft Mineralogy installed. Furthermore, the texture replacements are done via a built-in resourcepack. To revert the textures or use different ones, simply install one of the many amazing resourcepacks available for Minecraft 1.8

Joining a server with a different set of mods does not throw an error because Minecraft-Forge doesn't know which of your mods are client-only and which ones should be on both the client and server. You should create a new Minecraft profile for that server with correctly matching mods or use a launcher that automatically handles mod installation for you.

In short, Forge (and Minecraft Mineralogy) are working as intended and joining a server with a different set of mods than the client is considered an error on the part of the player.

Also, Mineralogy does _not_ use the Ore Renderer mod or any other mods besides Forge itself.

**Comment by AlvarOsO** [comment]
Isn't there a posibility to make a config option to disable the resource pack?

**Comment by cyanobacterium** [comment]
Nope. That's baked into the core of Minecraft, but you can easily take the assets folder from your vanilla Minecraft install and turn it into a resourcepack to prevent any mod from replacing the vanilla models.
